docs/developer-guide/release.md

```diff
@@ -145,4 +145,5 @@ Notify the project manager that the release should be announced on Mastodon.
 ### Email Mondo Users
 Send an email to Mondo users: mondo-users@googlegroups.com  
 (Note - the recipients usually a picture of someone's pet to be included.)
+1. Subject: The new Mondo release is available
```

Comment:
Do you want to include the month of the release in the email subject, e.g. `The <INSERT-MONTH> Mondo release is available`?

Reply:
sure. updated.